custom.css

```diff
@@ -0,0 +1,28 @@
+.info {
+    margin: 20px;
+    padding: 0 20px;
+    border-inline-start: 2px solid var(--links);
+}
+
+.info:before {
+    position: absolute;
+    width: 3rem;
+    height: 3rem;
+    margin-inline-start: calc(-1.5rem - 21px);
+    content: "ⓘ";
+    text-align: center;
+    background-color: var(--bg);
+    color: var(--links);
+    font-weight: bold;
+    font-size: 2rem;
+}
+
+/*
+  Make prompts unselectable in console sections.
+  This relies on highlight.js applying certain classes on the prompts.
+  For more details, see https://highlightjs.readthedocs.io/en/latest/css-classes-reference.html#stylable-scopes
+*/
+.hljs-meta.prompt_ {
```

Comment:
I actually already solved it (by removing `.prompt_`), in a commit that doesn't exist in your force-push (e82be2c96e7b98d9521af006932df271413e2de0). Is there a reason for the force-push, rather than adding commits? I suggest that I restore the original branch, and then rebase your added commits on top of it. Is that OK with you?

Reply:
Updating highlight.js fixed it. I have force pushed because I have squashed some of the commits.

Reply:
Updated again in e7a4eec962ed32551d34d5db19404051ed23b14b.